docs/outlook/get-and-set-recurrence.md

```diff
@@ -1,40 +1,44 @@
 ---
-title: Get and set recurrence in an Outlook add-in
-description: This topic shows you how to use the Office JavaScript API to get and set various recurrence properties of an item in an Outlook add-in.
-ms.date: 07/08/2022
+title: Get and set the recurrence of appointments
+description: This topic shows you how to use the Office JavaScript API to get and set various recurrence properties of an appointment using an Outlook add-in.
+ms.date: 05/30/2024
 ms.topic: how-to
 ms.localizationpriority: medium
 ---
 
-# Get and set recurrence
+# Get and set the recurrence of appointments
 
-Sometimes you need to create and update a recurring appointment, such as a weekly status meeting for a team project or a yearly birthday reminder. You can use the Office JavaScript API to manage the recurrence patterns of an appointment series in your add-in.
+Sometimes you need to create and update a recurring appointment, such as a weekly status meeting for a team project or a yearly birthday reminder. Use the Office JavaScript API to easily manage the recurrence patterns of an appointment series in your add-in.
 
 > [!NOTE]
 > Support for this feature was introduced in [requirement set 1.7](/javascript/api/requirement-sets/outlook/requirement-set-1.7/outlook-requirement-set-1.7). See [clients and platforms](/javascript/api/requirement-sets/outlook/outlook-api-requirement-sets#requirement-sets-supported-by-exchange-servers-and-outlook-clients) that support this requirement set.
 
-## Available recurrence patterns
+## Review available recurrence patterns
```

Comment:
What does "review" mean? Is this material being repeated from another source?

Reply:
I wanted to make the header consistent with the rest of the headers in the article by starting it with a verb. However, now that I'm reading it again, adding "review" does seem out of place. Changing this to "Recurrence patterns".